apps/daimo-mobile/src/action/useExistingAccount.ts

```diff
@@ -29,6 +30,8 @@ export function useExistingAccount(
 
   // TODO: Does TRPC have a better way to "watch" an endpoint?
   useEffect(() => {
+    if (alreadyCreating) return; // Disable use existing check if already creating
```

Comment:
guessing this is the race condition fix?

Reply:
yep